contracts/src/v2/Types.sol

```diff
@@ -26,7 +26,6 @@ library CommandKind {
     uint8 constant UnlockNativeToken = 2;
     uint8 constant RegisterForeignToken = 3;
     uint8 constant MintForeignToken = 4;
-    uint8 constant CreateAgent = 5;
     uint8 constant CallContract = 6;
```

Comment:
```suggestion
    uint8 constant CallContract = 5;
```

Also please make sure the substrate side matches the above and also doesn't have any remaining traces of `Command::CreateAgent`

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/21